modules.d/01systemd-hostnamed/module-setup.sh

```diff
@@ -44,6 +44,6 @@ install() {
         inst_multiple -H -o \
             /etc/hostname \
             "$systemdsystemconfdir"/systemd-hostnamed.service \
-            "$systemdsystemconfdir"/systemd-hostnamed.service.d/*.conf"
+            "$systemdsystemconfdir"/systemd-hostnamed.service.d/*.conf
```

Comment:
Interesting I fixed this last night but apparently forgot to push it, anyway fix it to look like this 
` "$systemdsystemconfdir/systemd-hostnamed.service.d/*.conf" `
since the `dir"/systemd` quote is the wrong one

Reply:
Thanks for the fix.